tools/proto-js-plugin/src/main/java/io/spine/js/generate/CodeLines.java

```diff
@@ -160,14 +198,14 @@ public void enterMethod(String methodName, String... methodArgs) {
         checkNotNull(methodArgs);
         String argString = join(", ", methodArgs);
         addLine(methodName + " = function(" + argString + ") {");
-        currentDepth++;
+        increaseDepth();
     }
 
     /**
      * Exits method declaration.
      */
-    public void exitFunction() {
-        currentDepth--;
+    public void exitMethod() {
+        decreaseDepth();
```

Comment:
There might be an improvement done to the depth processing. For example, we may define the depth concept as a class and incorporate it with `Indent` class instead of manually computing the number of spaces to print.

Also, is it necessary that `increaseDepth()` and `decreaseDepth()` method are `public`?